lib/book_parser.py

```diff
@@ -106,19 +104,26 @@ def _get_paragraph_data(self, codec):
 
         # find all paragraphs (sections enclosed in html p tags) and their starting offset
         for node in re.finditer(PARAGRAPH_PAT, book_html):
-            # get plain text from paragraph and locations of letters from beginning of file
-            results = [(word.group(0)[1:-1].decode(codec), word.start(0))
-                       for word in re.finditer(PLAIN_TEXT_PAT, node.group(0))]
             word_loc = {'words': '', 'locs': [], 'char_sizes': []}
```

Comment:
What is this fixing?

I assume there is something going on here with length offsets where HTML tags are included..  Are you just trying to exclude any HTML tags from the word lists?